src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -299,6 +344,13 @@ private void toggleClimbingSim() {
     Logger.recordOutput("Climber/Climbing", isClimbingStateSim);
   }
 
+  private Command emergencyUndo() {
```

Comment:
I think retract and emergencyUndo should be the same Command. Either way we want to make sure the servos are disengaged before moving.